webpack.config.js

```diff
@@ -134,13 +133,13 @@ module.exports = function(env) {
 			}),
 			new CopyWebpackPlugin([
 				{
-					from: './src/migrations',
+					from: './build/migrations',
 					to: 'migrations',
 				},
 			]),
 			new webpack.ContextReplacementPlugin(
 				/\.\/migrations/,
-				path.resolve(__dirname, 'src/migrations')
+				path.resolve(__dirname, 'build/migrations')
```

Comment:
@CameronDiver would you be able to test the migrations work in a minified build please - this change is probably right but the migrations are a bit of black magic so I'm not certain